docs/user/02-logs.md

```diff
@@ -0,0 +1,459 @@
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+    - custom: |
+        Name    grep
+        Regex   $kubernetes['labels']['app'] my-deployment
+    - custom: |
+        Name    grep
+        Exclude $kubernetes['namespace_name'] kyma-system|kube-system|istio-system
+    - custom: |
+        Name    record_modifier
+        Record  cluster_identifier ${KUBERNETES_SERVICE_HOST}
+  input:
+    ...
+  output:
+    ...
+```
+> **NOTE:** If you use a `custom` output, you put the LogPipeline in the [unsupported mode](#unsupported-mode).
+
+ The Telemetry Operator supports different types of [Fluent Bit filter](https://docs.fluentbit.io/manual/concepts/data-pipeline/filter). The example uses the [grep](https://docs.fluentbit.io/manual/pipeline/filters/grep) and the [record_modifier](https://docs.fluentbit.io/manual/pipeline/filters/record-modifier) filter.
+
+- The first filter keeps all log records that have the `kubernetes.labels.app` attribute set with the value `my-deployment`; all other logs are discarded. The `kubernetes` attribute is available for every log record. See [Kubernetes filter (metadata)](#kubernetes-filter-metadata) for more details.
+- The second filter drops all log records fulfilling the given rule. Here, typical Namespaces are dropped based on the `kubernetes` attribute.
+- A log record is modified by adding a new attribute. Here, a constant attribute is added to every log record to record the actual cluster node name at the record for later filtering in the backend system. As a value, a placeholder is used referring to a Kubernetes-specific environment variable.
+
+### Step 4: Add authentication details from Secrets
+
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets.
+
+Using the **http** output definition and the **valueFrom** attribute, you can map Secret keys as in the following **http** output example:
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  output:
+     http:
+        dedot: false
+        port: "80"
+        uri: "/"
+        host:
+           valueFrom:
+              secretKeyRef:
+                 name: http-backend-credentials
+                 namespace: default
+                 key: HTTP_ENDPOINT
+        user:
+           valueFrom:
+              secretKeyRef:
+                 name: http-backend-credentials
+                 namespace: default
+                 key: HTTP_USER
+        password:
+           valueFrom:
+              secretKeyRef:
+                 name: http-backend-credentials
+                 namespace: default
+                 key: HTTP_PASSWORD
+  input:
+    ...
+  filters:
+    ...
+```
+
+
+The related Secret must fulfill the referenced name and Namespace, and contain the mapped key as in the following example:
```

Comment:
"fulfil" or "fill out"?

Reply:
the thingy equal to "satisfy". The secret must have that name.